### DISCUSSION TITLE
Почему запрещен TS?

### DISCUSSION THREAD

**Discussion by TNikolay** [rolling-scopes-school/tasks#1019] [General]
https://github.com/rolling-scopes-school/tasks/blob/master/tasks/songbird/songbird-2022q3.md
"запрещено использовать TypeScript" (c)
Извиняюсь что беспокою, но очень интересно почему вдруг?
Ладно JQuery или какие-то библиотеки облегчающие жизнь - ок, но TS на таком задании только усложнит все, почему нельзя?
ортодоксы:)

**Comment by WiiJoy** [comment]
TS - тема stage#2. На первом все выполняется на чистом JS
